doc/api/http.md

```diff
@@ -917,6 +917,64 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaderNames()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing the unique names of the current outgoing headers.
```

Comment:
Yes, because header names can be repeated in the outoing headers, which could cause duplicate values in this Array, but it doesn't, the Array can be relied on to be unique.